sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -923,12 +923,12 @@ def init_port_sfp_status_tbl(port_mapping, xcvr_table_helper, stop_event=threadi
 def is_fast_reboot_enabled():
     fastboot_enabled = False
     state_db_host =  daemon_base.db_connect("STATE_DB")
-    fastboot_tbl = swsscommon.Table(state_db_host, 'FAST_REBOOT')
+    fastboot_tbl = swsscommon.Table(state_db_host, 'FAST_RESTART_ENABLE_TABLE')
     keys = fastboot_tbl.getKeys()
 
     if "system" in keys:
-        output = subprocess.check_output('sonic-db-cli STATE_DB get "FAST_REBOOT|system"', shell=True, universal_newlines=True)
-        if "1" in output:
+        output = subprocess.check_output(['sonic-db-cli', 'STATE_DB', 'hget', "FAST_RESTART_ENABLE_TABLE|system", 'enable'], universal_newlines=True)
```

Comment:
This PR is removing ` shell=True,` option. Please verify if that's intentional or merely due to copying a similar logic from master where ` shell=True,` is not used originally.

Reply:
Modified, thanks.